### PR TITLE
To show or not to show the tooltip.

### PR DESCRIPTION
**Pull request by RamonGilabert**
That's the question!

### CODE REVIEW

File: Demos/Basic-ObjC/Basic-ObjC/Assets/forms.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
               "title":"First name",
               "subtitle":"Fornavn",
               "type":"name",
+              "disabled": "YES",
```
Comment (zenangst) [inline Demos/Basic-ObjC/Basic-ObjC/Assets/forms.json]: This should be `true`.

File: Source/Cells/Text/FORMTextFieldCell.m [hunk 2]
```diff
@@ -35,6 +36,11 @@ - (instancetype)initWithFrame:(CGRect)frame
     UITapGestureRecognizer *tapGestureRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(cellTapAction)];
     [self addGestureRecognizer:tapGestureRecognizer];
 
+    [[NSNotificationCenter defaultCenter] addObserver:self
```
Comment (zenangst) [inline Source/Cells/Text/FORMTextFieldCell.m]: Where do you remove the observer?

Comment (RamonGilabert) [inline Source/Cells/Text/FORMTextFieldCell.m]: I was removing them now, I forgot, it should be fixed in the next commit.

File: Demos/Basic-ObjC/Basic-ObjC/Assets/forms.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
               "title":"First name",
               "subtitle":"Fornavn",
               "type":"name",
+              "disabled": "true",
```
Comment (zenangst) [inline Demos/Basic-ObjC/Basic-ObjC/Assets/forms.json]: Lose the quotes

```
disabled: true
```

File: Source/FORMData.m [hunk 4]
```diff
@@ -45,6 +45,8 @@ - (instancetype)initWithJSON:(id)JSON
               disabledFieldsIDs:disabledFieldIDs
                        disabled:disabled];
 
+
```
Comment (zenangst) [inline Source/FORMData.m]: Why these newlines? :smile:

Comment (RamonGilabert) [inline Source/FORMData.m]: That wasn't me! :o

**Comment by RamonGilabert** [comment]
All is well already! :)

File: Source/FORMData.m [hunk 5]
```diff
@@ -875,11 +875,13 @@ - (NSArray *)updateTargets:(NSArray *)targets
 
 - (NSArray *)enableTargets:(NSArray *)targets
 {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips object:[NSNumber numberWithBool:YES]];
```
Comment (NSElvis) [inline Source/FORMData.m]: Do you mind using a literals here? 

`@YES` => `[NSNumber numberWithBool:YES]`

Method formatting `:` goes in newlines
```objc
[[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips
                                                    object:@YES];
```

File: Source/FORMData.m [hunk 6]
```diff
@@ -875,11 +875,13 @@ - (NSArray *)updateTargets:(NSArray *)targets
...
     return [self updateTargets:targets withEnabled:YES];
 }
 
 - (NSArray *)disableTargets:(NSArray *)targets
 {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips object:[NSNumber numberWithBool:NO]];
```
Comment (NSElvis) [inline Source/FORMData.m]: Do you mind using a literals here? 

`@NO` => `[NSNumber numberWithBool:NO]`

Method formatting `:` goes in newlines
```objc
[[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips
                                                    object:@NO];
```

**Pull request merged**